src/main/java/application/controller/DeleteUserController.java

```diff
@@ -9,16 +10,19 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/delete")
+@WebServlet("/user/delete")
 public class DeleteUserController extends HttpServlet {
     private static final Injector injector = Injector.getInstance("application");
     private UserService userService = (UserService) injector.getInstance(UserService.class);
+    private ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         Long id = Long.valueOf(req.getParameter("id"));
         userService.delete(id);
+        shoppingCartService.delete(id);
```

Comment:
we remove ShoppingCart by userId, but method delete() expect shoppingCartId

Reply:
done